composer.json

```diff
@@ -17,7 +17,7 @@
     ],
     "require": {
         "php": ">=5.5.9",
-        "doctrine/common": "~2.3",
+        "doctrine/common": "~2.4",
```

Comment:
I'm totally fine with bumping the Doctrine base versions btw. 2.2 and 2.3 are long EOLed.